binary_search_tree/BinarySearchTree.py

```diff
@@ -0,0 +1,125 @@
+class Node(object):
+# Node for Binary search Tree and set the Node
+    def __init__(self, data):
+        self.data = data
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmphwomd5ps/binary_search_tree/BinarySearchTree.py
+++ b/tmp/tmphwomd5ps/binary_search_tree/BinarySearchTree.py
@@ -1,5 +1,5 @@
 class Node(object):
-# Node for Binary search Tree and set the Node
+    # Node for Binary search Tree and set the Node
     def __init__(self, data):
         self.data = data
         self.left = None
```